jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/installations/InstallationRegistrationEndpoint.java

```diff
@@ -217,17 +223,28 @@ public Response registerInstallation(
     @Produces(MediaType.APPLICATION_JSON)
     @ReturnType("org.jboss.aerogear.unifiedpush.rest.EmptyJSON")
     public Response increasePushMessageReadCounter(@PathParam("id") String pushMessageId,
-                           @Context HttpServletRequest request) {
+                           @Context HttpServletRequest request) throws IOException {
 
         // find the matching variation:
         final Variant variant = loadVariantWhenAuthorized(request);
         if (variant == null) {
             return create401Response(request);
         }
 
-        //let's do update the analytics
         if (pushMessageId != null) {
+            try (InputStream props = this.getClass().getResourceAsStream("/kafka/producer.props")) {
```

Comment:
I think for now embedding resource bundles, w/ hard-coded values are OK.

I'd like to move body of code out to a helper class, e.g. in the `common` module.